ui/lib/core/addon/components/modal.js

```diff
@@ -26,7 +26,7 @@ export default Component.extend({
   glyph: computed('type', function() {
     const modalType = this.get('type');
     if (!modalType) {
-      return {};
+      return;
```

Comment:
this was to fix an empty icon element for the models (like this one and the transit-key-actions modal) that did not have an icon.